### PR TITLE
Add locale

### PR DESCRIPTION
**Pull request by quentinleroy**

### CODE REVIEW

File: source/includes/_bookings.md [hunk 1]
```diff
@@ -695,7 +696,7 @@ You can now retrieve a list of all your booking by calling the [bookings#index](
 
 ### Checkout Link
 
-`GET https://belladonna.distribusion.com/bookings/checkout`
+`GET https://bustickets.distribusion.com/bookings/checkout`
```
Comment (dasnotme) [inline source/includes/_bookings.md]: - [ ] Better be consistent and use the `https://bustickets-demo.distribusion.com` domain.

File: source/includes/_bookings.md [hunk 2]
```diff
@@ -1112,7 +1113,7 @@ Error Code  | Meaning
 ----------- | -------
 400.000.000 | The provided json is invalid
 400.100.000 | Request invalid
-400.100.041 | Booking is invalid or missing
+400.101.041 | Booking is invalid or missing
```
Comment (dasnotme) [inline source/includes/_bookings.md]: - [ ] Breaking change.

File: source/includes/_bookings.md [hunk 3]
```diff
@@ -1141,7 +1142,7 @@ Error Code  | Meaning
...
```
Comment (dasnotme) [inline source/includes/_bookings.md]: - [ ] Breaking change.

File: source/includes/_connections.md [hunk 4]
```diff
@@ -3,9 +3,10 @@
 ## Find
 
 ```shell
-curl -X GET \
-  'https://api-demo.distribusion.com/retailers/v4/connections/find?departure_stations%5B%5D=FRLILBDT&arrival_stations%5B%5D=FRLYSPER&departure_date=2017-09-29&pax=1&locale=en&currency=EUR' \
+curl -g -X GET \
+  'https://api-demo.distribusion.com/retailers/v4/connections/find?departure_stations[]=FRLILBDT&departure_stations[]=FRLILCSA&departure_stations[]=FRLILGEU&departure_stations[]=FRLILGFA&departure_stations[]=FRLILZOB&arrival_stations[]=FRLYSAIR&arrival_stations[]=FRLYSDAR&arrival_stations[]=FRLYSIHO&arrival_stations[]=FRLYSPDB&arrival_stations[]=FRLYSPER&arrival_stations[]=FRLYSZOB&departure_date=2017-11-29&pax=1&locale=en&currency=EUR' \
```
Comment (dasnotme) [inline source/includes/_connections.md]: - [ ] Should we change the departure date to something in the future?

File: source/includes/_errors.md [hunk 5]
```diff
@@ -54,12 +54,14 @@ Error Code  | Meaning
 400.101.038 | Extra quantity is invalid or missing
 400.101.039 | Marketing carrier booking number is invalid or missing
 400.101.040 | Marketing carrier is invalid or missing
-400.100.041 | Booking is invalid or missing
+400.101.041 | Booking is invalid or missing
```
Comment (dasnotme) [inline source/includes/_errors.md]: - [ ] Breaking change.

File: source/includes/_bookings.md (hunk 1 above)
Comment (quentinleroy) [reply source/includes/_bookings.md]: Good point, will change this.

File: source/includes/_bookings.md (hunk 2 above)
Comment (quentinleroy) [reply source/includes/_bookings.md]: Its not a breaking change, it is the wrong error code in the master, which I am correcting

File: source/includes/_bookings.md (hunk 3 above)
Comment (quentinleroy) [reply source/includes/_bookings.md]: Its not a breaking change, it is the wrong error code in the master, which I am correcting

File: source/includes/_connections.md (hunk 4 above)
Comment (quentinleroy) [reply source/includes/_connections.md]: agree - will do so

File: source/includes/_errors.md (hunk 5 above)
Comment (quentinleroy) [reply source/includes/_errors.md]: Same as above

File: source/includes/_bookings.md (hunk 1 above)
Comment (quentinleroy) [reply source/includes/_bookings.md]: ah no - in all the documentation it is that way, the examples are with demo, but the endpoint in the documentation is the production one. Let's keep it that way for consistency with the overall documentation.

File: source/includes/_bookings.md (hunk 2 above)
Comment (dasnotme) [reply source/includes/_bookings.md]: might be, but isn't this documentation already online and being used by our partners?

**Pull request merged**